src/components/SQFormScrollableCard/SQFormScrollableCard.tsx

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
-import {
-  Card,
-  CardHeader,
-  CardContent,
-  CardActions,
-  Grid,
-  makeStyles,
-} from '@material-ui/core';
+import {Card, CardHeader, CardContent, CardActions, Grid} from '@mui/material';
 import {Formik, Form} from 'formik';
 import {useDebouncedCallback} from 'use-debounce';
 import SQFormButton from '../SQForm/SQFormButton';
 import SQFormHelperText from '../SQForm/SQFormHelperText';
 import {useInitialRequiredErrors} from '../../hooks/useInitialRequiredErrors';
 import {HEADER_HEIGHT} from '../../utils/constants';
-import type {CreateCSSProperties} from '@material-ui/core/styles/withStyles';
-import type {TypographyVariant, GridProps} from '@material-ui/core';
+import type {GridProps} from '@mui/material';
 import type {FormikHelpers, FormikValues} from 'formik';
 import type {AnyObjectSchema} from 'yup';
 
 export type SQFormScrollableCardProps<Values extends FormikValues> = {
-  /** An object of css-in-js style properties to be passed and spread onto `classes.cardContent` */
-  cardContentStyles?: CreateCSSProperties;
+  /** An object of css-in-js style properties to be passed and spread onto the `CardContent` component */
+  cardContentStyles?: Record<string, unknown>;
```

Comment:
We should be slight more lenient with our typing here. I think we should crib exactly whatever the `sx` prop is. I think it also accepts an array? Is there a particular reason some variation of CSS PRoperties wouldn't work?

Reply:
@20BBrown14 I am working on trying to implement @laurelbean's suggestion.  I couldn't find the `CreateCSSProperties` type in MUI5 which is the reason for the change.

Reply:
@laurelbean @20BBrown14 Here is an updated loom outlining the solution I came to and why I went that direction.  Feel free to take a look if interested. https://www.loom.com/share/41cc6dc460994552ad3354c56c3c9690